### DISCUSSION TITLE
Copernicus login credentials

### DISCUSSION THREAD

**Discussion by Matthew-Wilson-49** [semiautomaticgit/SemiAutomaticClassificationPlugin#385] [Q&A] [answered]
Hello! When trying to download satellite images, I always get HTTP error 401 since SCP fails to fetch my Copernicus credentials. SCP only prompts you for User and Password, but Copernicus doesn't have an explicit user name. It's got User ID and Account ID, along with the ability to make OAuth credentials, so I'm not sure what credentials to input into SCP. Thanks in advance!

**Comment by semiautomaticgit** [comment, marked as answer]
Hello @Matthew-Wilson-49 ,

for Copernicus services the user is the email of registration.
In case of issues, you can select the product Sentinel-2_MPC (Sentinel-2 images from the Microsoft Planetary Computer ) that doesn't require any credential for search and download.

**Reply by Matthew-Wilson-49** [reply]
Thank you very much, this worked!
